CHANGES.txt

```diff
@@ -1,4 +1,43 @@
-﻿NUnit 3.6.1 - March 6, 2017
+﻿NUnit 3.7 - July 13, 2017
```

Comment:
That would be a good idea. Unfortunately the release is out, but we could update it for the next release. I would like to switch this to a markdown file too, like we did for the framework.